client/src/components/Workflow/Editor/Node.vue

```diff
@@ -245,6 +247,14 @@ const isEnabled = getGalaxyInstance().config.enable_tool_recommendations; // get
 
 const isActive = computed(() => props.id === props.activeNodeId);
 
+const isPopulatedInput = computed(
+    () =>
+        props.populatedInputs &&
+        isWorkflowInput(props.step.type) &&
+        "nodeText" in props.step &&
```

Comment:
I think a better way to handle this is to split the component into one that knows about inputs and one that doesn't. I think as a general pattern that would also avoid the v-if/v-else-if/v-else djungle in so many of our components. Keep it in mind for the future.